app/controllers/api/base_controller/parser.rb

```diff
@@ -299,7 +299,7 @@ def validate_post_api_action_as_subcollection(cname, mname, aname)
         return if cname == @req.collection
         return if collection_config.subcollection_denied?(@req.collection, cname)
 
-        aspec = collection_config.typed_subcollection_actions(@req.collection, cname)
+        aspec = collection_config.typed_subcollection_actions(@req.collection, cname, @req.s_id ? :subresource : :subcollection)
```

Comment:
Got it now, problem is that it worked for delete because it used the delete action's identifier on the subcollection, but we don't implement a bulk revert action on the snapshots subcollection (does not make sense to do so), thus this problem emerged.  So I'm good with this fix, Thanks.